src/resolvers/exotics/bitbucket-resolver.js

```diff
@@ -29,4 +43,16 @@ export default class BitbucketResolver extends HostedGitResolver {
   static getHTTPFileUrl(parts: ExplodedFragment, filename: string, commit: string): string {
     return `https://${this.hostname}/${parts.user}/${parts.repo}/raw/${commit}/${filename}`;
   }
+
+  async hasHTTPCapability(url: string): Promise<boolean> {
+    return (
+      (await this.config.requestManager.request({
```

Comment:
Storing the return value of this request in a nicely-named variable to give a hint to the reader about what this is and why are we doing things would be great. It would also make the `return` line more apparent:
```
// We don't follow redirects and reject a 302 since this means BitBucket won't allow us to use the HTTP protocol for `git` access. Most probably a private repo and this 302 is to a login page.
const bitbucketHTTPSupport = await ...;
return bitbucketHTTPSupport !== false;